algorithms/CPlusPlus/Arrays/remove-duplicates.cpp

```diff
@@ -0,0 +1,62 @@
+/*CPP program to remove the duplicates elements from the array in place
+  without using any extra space with o(1) extra memory 
+  Its impossible to resize the array in c++,(The only way possible is to allocate 
+  a new array copy the required values and delete the first one. and the requirement of given problem
+  was not to use any extra space so we are just displaying array upto which last element
```

Comment:
```suggestion
  was not to use any extra space so we are just displaying array up to which last element
```
Changes need here because code spell fail means your spelling was wrong correct it as shown above

Reply:
Sure I am doing that

Reply:
If you can check now